cmd/wstail/tail.go

```diff
@@ -273,6 +275,26 @@ func serveWs(w http.ResponseWriter, r *http.Request) (err error) {
 	return nil
 }
 
+// downloadLogs serves /v1/download
+//
+// it compresses the file with aggregated logs and pushes it to the response
+// stream
+func downloadLogs(w http.ResponseWriter, r *http.Request) error {
+	file, err := os.Open(filePath)
```

Comment:
yeah, I think it's about right - I'm not 100% sure, but it should be standard (`savelog`)